src/scripts/index.js

```diff
@@ -977,6 +977,10 @@ $(document).ready(function () {
 // timer
 $(document).ready(function () {
   var countDownDate = new Date('Dec 10, 2018 00:00:00').getTime()
+  function checkForLenght (_this) {
```

Comment:
2. Rename to `padDecimal`.
3. Rename `_this` to `number`.

Reply:
done